### DISCUSSION TITLE
Page Attribute Customize

### DISCUSSION THREAD

**Discussion by Hateeb** [zalify/easy-email-editor#90] [Q&A] [answered]
Have any method to Customize Page Attribute Panel Email Setting? If have any kindly suggest it me
https://i.imgur.com/xASKV26.png
I want add tooltip after subject label.

**Comment by m-Ryan** [comment, marked as answer]
https://github.com/arco-design/easy-email/blob/master/packages/easy-email-extensions/readme.md
```tsx
import { BlockAttributeConfigurationManager } from "easy-email-extensions";

BlockAttributeConfigurationManager.add({
  [BasicType.PAGE]: () => <div>will be overwrite `Page`</div>,
});

```

**Reply by Hateeb** [reply]
I do not overwrite complete page. I want only add tooltip or hepling text after subject
Check screen shot https://i.imgur.com/NnerXxK.png

**Reply by m-Ryan** [reply]
Relatived: https://github.com/arco-design/easy-email/blob/master/packages/easy-email-extensions/src/AttributePanel/components/blocks/Page/index.tsx
You can only override the whole page, but it won't be difficult, all the components you need are exported.
```tsx
import {
  ColorPickerField,
  InputWithUnitField,
  SwitchField,
  TextAreaField,
  TextField,
} from 'easy-email-extensions';

```

**Reply by Hateeb** [reply]
how to import AddFont and FontFamily

**Reply by m-Ryan** [reply]
```
import {
FontFamily
} from 'easy-email-extensions';
```
`AddFont ` is not exported, you can copy this file `https://github.com/arco-design/easy-email/blob/master/packages/easy-email-extensions/src/components/Form/AddFont.tsx`

**Reply by Hateeb** [reply]
Thanks For your time and help
